src/nih_wayfinding/app/scripts/routing/directions-service.js

```diff
@@ -130,6 +132,38 @@
         }
 
         /**
+         * Get a summary for a route, including time, distance and # of turns
```

Comment:
This data comes through on the `leg` object, including the total distance and start and end time, so it's not necessary to calculate it from the individual steps.